jwst/lib/set_telescope_pointing.py

```diff
@@ -325,10 +330,17 @@ def update_wcs_from_fgs_guiding(model, default_pa_v3=0.0, default_vparity=1):
         )
         vparity = default_vparity
 
+    try:
+        v3i_yang = model.meta.wcsinfo.v3yangle
+    except AttributeError:
+        logger.warning(f'Keyword "V3I_YANG" not found. Using {default_v3yangle} as default value.')
+
+        v3i_yang = default_v3yangle
+
     (model.meta.wcsinfo.pc1_1,
      model.meta.wcsinfo.pc1_2,
      model.meta.wcsinfo.pc2_1,
-     model.meta.wcsinfo.pc2_2) = calc_rotation_matrix(pa_rad, vparity=vparity)
+     model.meta.wcsinfo.pc2_2) = calc_rotation_matrix(pa_rad, v3i_yang, vparity=vparity)
```

Comment:
`v3i_yang` also needs to be converted to radians before being sent along to `calc_rotation_matrix()`